ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.15__jenkins-ocp4.15-lp-interop.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.13
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
```

Comment:
Check that openshift version is correct

Reply:
https://github.com/openshift/release/pull/44988#discussion_r1376808230

Reply:
going to update and test this